action.yml

```diff
@@ -5,207 +5,30 @@
 #  \__,_|_| |_|\__,_|_|\__, /___\___|
 #                     |___/
 ---
-name: 'Analyze PR'
-description: 'Analyze Pull request'
+name: Analyze PR
+author: Phylum, Inc. <engineering@phylum.io>
+description: Analyze pull request with Phylum
+branding:
+  icon:  package
+  color: blue
 inputs:
-  vul_threshold:
-    description: "Vulnerability risk score threshold value"
-    required: true
-  mal_threshold:
-    description: "Malicious Code risk score threshold value"
-    required: true
-  eng_threshold:
-    description: "Engineering risk score threshold value"
-    required: true
-  lic_threshold:
-    description: "License risk score threshold value"
-    required: true
-  aut_threshold:
-    description: "Author risk score threshold value"
+  github_token:
+    description: GITHUB_TOKEN with at least write access to pull-requests scope or a repo scoped PAT
     required: true
+    default: ${{ github.token }}
   phylum_token:
-    description: "Phylum token"
+    description: Phylum token
+    required: true
+  cmd:
+    description: The phylum-ci command line to pass to the Docker container's entrypoint
     required: true
-  phylum_version:
-    description: "Phylum version"
-    required: false
-    default: '0'
-  incomplete_package_strategy:
-    description: "Method for resolving incomplete packages"
-    required: false
-    default: "pass_with_comment"
-  invoke_test_matrix:
-    description: "Only used for testing"
-    required: false
-    default: false
-
-
+    default: phylum-ci --help
```

Comment:
Why is `--help` the default? It seems like the default should be to actually do the analysis.

Reply:
That was done based on recommendation from the [GitHub documentation for Docker container actions](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsargs):

> If the action exposes a --help flag, or something similar, use that to make your action self-documenting.

However...upon a second reading of that page, it also says to "use the guidelines ordered by preference" and a higher preference is to:

> Use defaults that allow using the action without specifying any args.

So...this will be changed.